tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -1187,6 +1239,7 @@ class R__CLING_PTRCHECK(off) MeanHelper : public RActionImpl<MeanHelper> {
    std::vector<ULong64_t> fCounts;
    std::vector<double> fSums;
    std::vector<double> fPartialMeans;
+   std::vector<double> fCompensations;
```

Comment:
(Why) do we need a global variable `fCompensations`? Can we pass it as an extra (local) argument to the functions that require it?

Reply:
Yes, it should be possible to pass it as an extra argument in void Exec(unsigned int slot, ResultType x), for instance.
Would that be better?

Reply:
okay, I think it is okay to remain like that, but see my latest comment. There was a place which did not initialize the elements of fCompensations!